window/src/os/macos/window.rs

```diff
@@ -1109,6 +1110,27 @@ impl WindowInner {
         }
     }
 
+    fn update_titlebar_background(&self, color: RgbaColor) {
+        unsafe {
+            let titlebar_view_container = get_titlebar_view_container(&self.window).unwrap();
```

Comment:
I really don't like bare `.unwrap()` calls, as each one represents a place where the program can either die or end up broken if the expectations aren't met.  I'd prefer for this to return without doing anything instead of panicking the program

Reply:
Switch this to an `if let` and added a log line on failure.